README.md

```diff
@@ -126,3 +126,14 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
```

Comment:
I'm looking for a README that, step by step, walks everyone through how to get the app set up on fly.io, both a production version AND a staging version.

A new developer should be able to follow all your instructions very simply, with no details missing

Reply:
Also nice would be explaining how to run it for free, so another developer can try it.

Reply:
ok